php/commands/src/CLI_Alias_Command.php

```diff
@@ -300,14 +300,17 @@ public function update( $args, $assoc_args ) {
 	/**
 	 * Get config path and aliases data based on config type.
 	 *
-	 * @param string $config Type of config to get data from.
-	 * @param string $alias  Alias to be used for Add/Update/Delete.
+	 * @param string $config             Type of config to get data from.
+	 * @param string $alias              Alias to be used for Add/Update/Delete.
+	 * @param bool $create_config_file Optional. If a config file doesn't exist,
+	 *                                 should it be created? Defaults to false.
```

Comment:
Oops, sorry, I messed up the alignment here (using the GitHub editor....):
```suggestion
	 * @param string $config             Type of config to get data from.
	 * @param string $alias              Alias to be used for Add/Update/Delete.
	 * @param bool   $create_config_file Optional. If a config file doesn't exist,
	 *                                   should it be created? Defaults to false.
```